datastore/builtindb.py

```diff
@@ -61,6 +61,8 @@ def insert(self, collection_name, data):
         except Exception as exception:
             raise exception
 
+        data.pop('_id', None)
```

Comment:
could you give a note, why we should do that?

```suggestion
        # pop `_id` to bla bla bla
        data.pop('_id', None)
```

Reply:
cool, on it